library/src/loader_impl.cxx

```diff
@@ -47,7 +47,8 @@ class loader_impl::internals
         progressData->timer->StopTimer();
         vtkProgressBarWidget* widget = progressData->widget;
         // Only show and render the progress bar if loading takes more than 0.15 seconds
-        if (progressData->timer->GetElapsedTime() > 0.15)
+        if (progressData->timer->GetElapsedTime() > 0.15 ||
+          std::getenv("CTEST_PROGRESS_BAR_TESTING") != nullptr)
```

Comment:
what about using `vtksys::SystemTools::GetEnv` instead of hiding `_CRT_SECURE_NO_WARNINGS` warnings?